shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -63,6 +63,8 @@
  */
```

Comment:
onSaveInstanceState is defined on android.view.View right? This only covers some of the components. What are you going to do for the FlutterView?

Reply:
If we use the view's onSaveInstanceState instead of the onSaveInstanceState in this file we would cover all components since the FlutterActivity/FlutterView just uses a View behind the scenes as well, right? Let me investigate what it would take to move this to the view.

Reply:
We discussed this in person and concluded that if you use a FlutterView directly, you'd have to hock this up manually.